prow/bugzilla/fake.go

```diff
@@ -175,7 +189,17 @@ func (c *Fake) CloneBug(bug *Bug) (int, error) {
 	if err != nil {
 		return 0, fmt.Errorf("failed to get parent bug's comments: %v", err)
 	}
-	return c.CreateBug(cloneBugStruct(bug, comments))
+	id, err := c.CreateBug(cloneBugStruct(bug, comments))
```

Comment:
Does this set the version correctly the first time around?

Reply:
Nope. Need to add that to the update var.